### PR TITLE
Remove junk and fixup configs

### PR DESCRIPTION
**Pull request by AdrianAndersen**
- Remove traces of flow
- move babel config to js
- delete unused stuff

Fixes ABA-149

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -1,16 +0,0 @@
-{
```
Comment (ivarnakken) [inline jsconfig.json:1]: Perhaps replace this with a tsconfig?

Comment (AdrianAndersen) [reply jsconfig.json:1]: We have a TSconfig already. This is just old junk

Comment (ivarnakken) [reply jsconfig.json:1]: ah yes, my bad

**Review by ivarnakken — APPROVED** [review]

**Review by eikhr — APPROVED** [review]

**Review by LudvigHz — CHANGES_REQUESTED** [review]
Do a `mv .babelrc babel.config.js` and commit that before you change the contents to preserve history :)

**Comment by AdrianAndersen** [comment]
Fixed, @LudvigHz

**Review by LudvigHz — APPROVED** [review]

**Pull request merged**